web-components/src/[sandbox]/examples/dropdown.ts

```diff
@@ -0,0 +1,152 @@
+import { DropdownEvents } from "@/components/dropdown/Dropdown";
+import { html } from "lit-element";
+
+export const dropdownStringOptions = ["one", "two", "three", "   non-trimmed "];
+
+export const dropdownStringLongOptions = [
+  "01",
+  "02",
+  "03",
+  "04",
+  "05",
+  "06",
+  "07",
+  "08",
+  "09",
+  "10",
+  "11",
+  "12",
+  "13",
+  "14",
+  "15",
+  "16",
+  "17",
+  "18",
+  "19",
+  "20",
+  "21",
+  "22",
+  "23",
+  "24",
+  "25",
+  "26",
+  "27",
+  "28",
+  "29",
+  "30",
+  "31",
+  "32",
+  "33",
+  "34",
+  "35",
+  "36",
+  "37",
+  "38",
+  "39",
+  "40"
+];
+
+export const dropdownObjectKeyValueOptions = [{ one: "One" }, { two: "Two" }, { three: "Three" }];
+
+export const dropdownObjectOptions = [
+  {
+    user_id: "xxx-111",
+    user_name: "jd",
+    full_name: "John Doe"
+  },
+  {
+    user_id: "xxx-222",
+    user_name: "as",
+    full_name: "Adam Smith"
+  },
+  {
+    user_id: "xxx-333",
+    user_name: "ba",
+    full_name: "Boris Anderson"
+  }
+];
+
+export const dropdownObjectLongOptions = [
+  { id: "0", country: "Afghanistan" },
+  { id: "1", country: "Aland Islands" },
+  { id: "2", country: "Albania" },
+  { id: "3", country: "Algeria" },
+  { id: "4", country: "American Samoa" },
+  { id: "5", country: "Andorra" },
+  { id: "6", country: "Angola" },
+  { id: "7", country: "Anguilla" },
+  { id: "8", country: "Antarctica" },
+  { id: "9", country: "Antigua and Barbuda" },
+  { id: "10", country: "Argentina" },
+  { id: "11", country: "Australia" },
+  { id: "12", country: "Austrian Empire" },
+  { id: "13", country: "Austria" },
+  { id: "14", country: "Azerbaijan" },
+  { id: "15", country: "Bahrain" },
+  { id: "16", country: "Bangladesh" },
+  { id: "17", country: "Barbados" },
+  { id: "18", country: "Bavaria" },
+  { id: "19", country: "Belarus" },
+  { id: "20", country: "Belgium" },
+  { id: "21", country: "Belize" },
+  { id: "22", country: "Cabo Verde" },
+  { id: "23", country: "Cambodia" },
+  { id: "24", country: "Cameroon" },
+  { id: "25", country: "Canada" },
+  { id: "26", country: "Cayman Islands" },
+  { id: "27", country: "The Central African Republic" },
+  { id: "28", country: "Chad" }
+];
+
+export const dropdownTemplate = html`
+  <h3>Strings Options</h3>
+  <md-dropdown
+    .options="${dropdownStringOptions}"
+    @dropdown-selected="${(e: CustomEvent<DropdownEvents["dropdown-selected"]>) => {
+      console.log("[Strings Options] Selected: ", e.detail);
+    }}"
+  ></md-dropdown>
+  <h3>Strings Options (Disabled)</h3>
+  <md-dropdown .options="${dropdownStringOptions}" disabled></md-dropdown>
```

Comment:
This disabled example still seems to be focusable, but without any styling. When I tab through all the dropdowns, the focus "dissapears" when it is on the disabled example. This is probably related to tabIndex.